browser/brave_profile_prefs.cc

```diff
@@ -211,6 +212,10 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       password_manager::prefs::kPasswordLeakDetectionEnabled,
       base::Value(false));
 
+  // clear default popular sites
```

Comment:
We should probably wrap it with `#if defined(OS_ANDROID)` as this option seems to be registered on android only.

Reply:
it is registered on all platforms as far as I can tell